doc/whats_new/v0.20.rst

```diff
@@ -175,6 +175,14 @@ Metrics
   :issue:`10093` by :user:`alexryndin <alexryndin>`
   and :user:`Hanmin Qin <qinhanmin2014>`.
 
+Feature Extraction
+
+- Fixed a bug in :func:`feature_extraction.image._compute_n_patches` where an exception
```

Comment:
you cannot point to a private function feature_extraction.image._compute_n_patches

make sure doc renders fine